### DISCUSSION TITLE
Welcome to plugin_weathermap Discussions!

### DISCUSSION THREAD

**Discussion by TheWitness** [thurban/plugin_weathermap#69] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TheWitness** [comment]
All, just have a few more bugs to work through.  Keep logging them and replying here or with your bug or feature requests in the Issue area.

**Comment by MSS970** [comment]
Hello,
I really appreciate your effort guys to enhance the weathermap plugin, this is really great, I just love it.
I will be posting my comments and bug finding.
Regards,
MSS

**Comment by max6491** [comment]
I would like to use the WeatherMaps with php 8 as standalone. is there a documentation that I can follow ?

**Comment by max6491** [comment]
I would like to use the WeatherMaps with php 8 as standalone. is there a documentation that I can follow ?

**Reply by TheWitness** [reply]
Sorry, it may be possible, but if it does not work, I don't plan on spending a whole lot of time fixing it.  Might as well run Cacti if for nothing else, to manage the maps.

**Comment by MSS970** [comment]
Hi,
The documentation is part of the plugin folder.

1. Download the weathermap plugin
2. Unzip the package
3. Rename the extracted folder as "weathermap"
4. Copy/Move the folder to /cacti/plugins
5. You may refer to the documentations under /cacti/plugins/weathermap/docs
6.  See the docs sub-directory for full HTML documentation, FAQ and example config.
7. You can start by clicking on index.html, then click on [New Users should start here.]

Let me know if you still need any further help.

**Reply by max6491** [reply]
I got the check.php working in CLI and web server, but I am getting below error.
PHP Warning:  require_once(Console/Getopt.php): Failed to open stream: No such file or directory in /var/www/html/weathermap on line 8
PHP Fatal error:  Uncaught Error: Failed opening required 'Console/Getopt.php' (include_path='.;/usr/bin/pear') in /var/www/html/weathermap:8
Stack trace:
#0 {main}
  thrown in /var/www/html/weathermap on line 8

**Reply by TheWitness** [reply]
This version of weather map really needs cacti to work. So if you're not going to use cacti you can still use the editor but you still need cacti because of all the embedded functions.

**Reply by TheWitness** [reply]
So what I'm trying to say is that you need cacti that doesn't mean you have to use cacti. I hope that's clear.

**Comment by MSS970** [comment]
Check if the required file exists in the specified location

**Reply by max6491** [reply]
already checked and can't find those files. are those supposed to be in the package ?
